modules/blocksruntime/1.0.0/MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "blocksruntime",
+    version = "1.0.0",
```

Comment:
Where are you getting this version number from? I couldn't find one in the repo.

Reply:
It's made up to reflect the fact that the actual code in the repo hasn't changed in more than a decade - I just didn't know what else to choose.

Reply:
Hmm I guess 6 years of non-change can also be considered "as stable as it gets" lol :rofl: Ok I think with this it's fine to leave this at 1.0.0.